src/main/java/fr/zcraft/quartzlib/components/gui/PromptGui.java

```diff
@@ -236,16 +273,29 @@ protected void open(final Player player) {
 
         RunTask.later(() -> {
             try {
-                final Object signTileEntity = fieldTileEntitySign.get(sign);
-                final Object playerEntity = methodGetHandle.invoke(player);
+                if (tileEntitySign == null) {
+                    final Object signTileEntity = fieldTileEntitySign.get(sign);
+                    final Object playerEntity = methodGetHandle.invoke(player);
+
+                    // In Minecraft 1.12+, there's a lock on the signs to avoid them
+                    // to be edited after they are loaded into the game.
+                    if (fieldTileEntitySignEditable != null) {
+                        fieldTileEntitySignEditable.set(signTileEntity, true);
+                    }
 
-                // In Minecraft 1.12+, there's a lock on the signs to avoid them
-                // to be edited after they are loaded into the game.
-                if (fieldTileEntitySignEditable != null) {
-                    fieldTileEntitySignEditable.set(signTileEntity, true);
-                }
+                    methodOpenSign.invoke(playerEntity, signTileEntity);
+                } else {
+                    //TODO utiliser tile entitysign methods here
```

Comment:
Still to do?

Reply:
No this was removed. that's very old support for 1.11- versions.